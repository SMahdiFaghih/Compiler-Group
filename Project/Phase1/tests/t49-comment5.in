
a comment block at the end /* of the line */
