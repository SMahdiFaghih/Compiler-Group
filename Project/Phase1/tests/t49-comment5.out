T_ID a
T_ID comment
T_ID block
T_ID at
T_ID the
T_ID end
